### DISCUSSION TITLE
原神

### DISCUSSION THREAD

**Discussion by Hjhnull** [share121/inter-knot#5302] [其他]
![1725693071518 png](https://github.com/user-attachments/assets/54e47e43-8176-4dc6-b9da-5153468149dd)
![1725692949864 png](https://github.com/user-attachments/assets/3cb70b32-9c41-4c12-8b00-aaf0ca153931)

**Comment by Loxic123** [comment]
豪堪<img src="https://upload-bbs.mihoyo.com/upload/2022/08/25/d83f447b53326fddfe39a5b53e4025b8_2689322449412256346.png" alt="爱莉希雅_心动" width="75" >
